packages/calcite-components/src/components/action-bar/action-bar.stories.ts

```diff
@@ -82,6 +82,7 @@ export const horizontal = (): string => html`
         <calcite-action text="Layers" icon="layers"> </calcite-action>
       </calcite-action-group>
       <calcite-action slot="actions-end" text="hello world" icon="layers"> </calcite-action>
+      <calcite-action slot="bottom-actions" text="hello world 2" icon="information"> </calcite-action>
```

Comment:
Can you add a comment somewhere around this to indicate that the slot is deprecated? Applies to the `horizontalSmall` story too.